src/main/scala/com/linkedin/feathr/offline/job/FeatureTransformation.scala

```diff
@@ -1,7 +1,9 @@
 package com.linkedin.feathr.offline.job
 
-import com.linkedin.feathr.common._
 import com.linkedin.feathr.common.exception.{ErrorLabel, FeathrException, FeathrFeatureTransformationException}
+import com.linkedin.feathr.common.tensor.TensorData
+import com.linkedin.feathr.common.types.FeatureType
+import com.linkedin.feathr.common.{AnchorExtractorBase, _}
```

Comment:
Are the changes in this file newly added? Or just being transferred over from the LinkedIn fork?

Reply:
There's minor refactor based off Li fork.